packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -312,7 +312,8 @@ export class StatsAnalyzer extends EventsScope {
         newMqa.audioReceive[1].streams.push(audioReceiverStream);
 
         this.lastMqaDataSent[mediaType].recv = cloneDeep(this.statsResults[mediaType].recv);
-      } else if (mediaType.includes('video-send')) {
+      } else if (mediaType.includes('video-send-layer')) {
```

Comment:
better to do startsWith?

Reply:
Updated, added sample output and screenshots to the description.